translations/views.py

```
@@ -48,31 +48,25 @@ def get(self, request):
-                error_response = {
-                    "error": "Translations temporarily unavailable",
-                    "error_type": type(retry_error).__name__,
-                    "message": str(retry_error),
-                }
-
-                # Add traceback in debug mode
-                if settings.DEBUG:
-                    error_response["traceback"] = traceback.format_exc()
-
-                return Response(error_response, status=503)
+        except Exception as error:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)